app/database.py

```
@@ -0,0 +1,20 @@
+import os
+from typing import AsyncGenerator
```

Ruff (0.8.2)

Import from `collections.abc` instead: `AsyncGenerator`

Import from `collections.abc`

(UP035)